frontend/src/app/prereq-tree/treepane/treepane.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="trees">
```

Comment:
Could you maybe make it scrollable from here?

Reply:
maybe try adding "overflow: scroll;" in css trees class?